### PR TITLE
fix assignments and refactoring

### PR DESCRIPTION
**Pull request by mcl-sz**
Fixed some assignments and some refactoring of var names.

### CODE REVIEW

File: src/Bielu.Examine.ElasticSearch/Services/ElasticsearchService.cs [hunk 1]
```diff
@@ -371,27 +371,31 @@ private BulkRequestDescriptor ToElasticSearchDocs(IEnumerable<ValueSet> docs, st
             }
             catch (Exception e)
             {
-#pragma warning disable CA1848
```
Comment (bielu) [inline src/Bielu.Examine.ElasticSearch/Services/ElasticsearchService.cs:374]: That is required to disable analyzer to not complain about performance, I have plan to switch it to other one but it is really complicated.

File: src/Bielu.Examine.Core/Indexers/ElasticSearchBaseIndex.cs [hunk 2]
```diff
@@ -23,7 +23,7 @@ public class ElasticSearchBaseIndex(
     private ExamineIndexState IndexState => indexStateService.GetIndexState(name);
     private static readonly object _existsLocker = new object();
 
-    private IDisposable Unsubscriber => elasticSearchService.Subscribe(this);
```
Comment (bielu) [inline src/Bielu.Examine.Core/Indexers/ElasticSearchBaseIndex.cs:26]: afaik it is required in Observer patter, but I am not 100% sure about it.

**Comment by mcl-sz** [comment]
The problem is that is wat complaining that Unsubscriber  was readonly. I could not compile it this way. In the Microsoft  example they declare it like this as well: https://learn.microsoft.com/en-us/dotnet/standard/events/how-to-implement-an-observer#example

File: src/Bielu.Examine.Core/Indexers/ElasticSearchBaseIndex.cs (hunk 2 above)
Comment (mcl-sz) [reply src/Bielu.Examine.Core/Indexers/ElasticSearchBaseIndex.cs:26]: The problem is that is wat complaining that Unsubscriber was readonly. I could not compile it this way. In the Microsoft example they declare it like this as well: https://learn.microsoft.com/en-us/dotnet/standard/events/how-to-implement-an-observer#example

File: src/Bielu.Examine.ElasticSearch/Services/ElasticsearchService.cs (hunk 1 above)
Comment (mcl-sz) [reply src/Bielu.Examine.ElasticSearch/Services/ElasticsearchService.cs:374]: ah ok

**Pull request merged**